UI/window-projector.cpp

```diff
@@ -333,33 +333,33 @@ void OBSProjector::OBSRenderMultiview(void *data, uint32_t cx, uint32_t cy)
 	auto calcBaseSource = [&](size_t i)
 	{
 		switch (multiviewLayout) {
-		case MultiviewLayout::VERTICAL_LEFT:
+		case MultiviewLayout::VERTICAL_LEFT_8_SCENES:
 			sourceX = halfCX;
 			sourceY = (i / 2 ) * quarterCY;
 			if (i % 2 != 0)
 				sourceX = halfCX + quarterCX;
 			break;
-		case MultiviewLayout::VERTICAL_RIGHT:
+		case MultiviewLayout::VERTICAL_RIGHT_8_SCENES:
 			sourceX = 0;
 			sourceY = (i / 2 ) * quarterCY;
 			if (i % 2 != 0)
 				sourceX = quarterCX;
 			break;
-		case MultiviewLayout::HORIZONTAL_BOTTOM:
+		case MultiviewLayout::HORIZONTAL_BOTTOM_8_SCENES:
 			if (i < 4) {
-				sourceX = (float(i) * quarterCX);
+				sourceX = i * quarterCX;
```

Comment:
I noticed you unnecessarily changing the explicit casts all over here.  I did explicit casts here to make the reader of the code more aware that the variable is not a float, and to prevent any conversion warnings (if any).  I admit it's become a habit due to microsoft's compiler design, but still, I feel these changes were an unnecessary/unrealted part of this commit/diff.